modules/backend/models/Preference.php

```diff
@@ -179,7 +179,8 @@ protected function getFallbackLocale($locale)
     public function getLocaleOptions()
     {
         $localeOptions = [
-            'be' => [Lang::get('system::lang.locale.be'), 'flag-by'],
+            'ar' => [Lang::get('system::lang.locale.ar'), 'flag-sa'],
+		    'be' => [Lang::get('system::lang.locale.be'), 'flag-by'],
```

Comment:
The indentation on this is off by one space to the left

Reply:
Actually, it looks like it's getting indented by tabs. Change all the indentation to only be spaces

Reply:
It looks good when you view the file or edit it. I removed tabs and used spaces, but it's the same!